capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -42,21 +53,71 @@ export class WikiMediaComponent implements OnChanges, OnInit {
   getResults(queryString: string) {
     const currentQuery = queryString;
     let isOldRequest = false;
-    this.wikiService.search(queryString)
+    this.wikiService.search(queryString, 'en', 'History')
       .subscribe((result: WikiServiceResult) => {
         if (result.history && currentQuery === this.cityName) {
-            isOldRequest = true;
-            this.body = result.history;
-            this.history = result.history;
-            this.urls = result.furtherReading;
-            this.title = result.title;
-            this.loading = false;
-            this.cd.detectChanges();
+          this.body = result.history;
+          this.history = result.history;
+          this.urls = result.furtherReading;
+          this.title = result.title;
+          this.loading = false;
+          this.langlinks = result.langlinks;
+          const language = this.langlinks.get('ca');
+          this.prevQuery = language.get('searchQuery');
+          this.prevWordForHistory = 'History';
+          this.prevPreFix = 'en';
+          this.cd.detectChanges();
         }
         else if (!isOldRequest) {
           this.error = 'API did not return a valid response.';
           console.error('API did not return a valid response.');
         }
     });
   }
+
+  changeLanguage(queryString: string, languagePrefix: string, wordForHistory: string) {
+    this.wikiService.searchNewLang(queryString, languagePrefix, wordForHistory)
+      .subscribe((result: WikiServiceResult) => {
+        this.history = result.history;
+        console.log(this.history);
+        if (!(this.history) || (this.history.includes("CPU time usage:"))){
+          this.loading = true;
+          this.error = "Unfortunately, the language you requested does not have an available tranlation for this page. Please select a different language.";
```

Comment:
Use single quotes. Also, break this up into two lines, as it's way to long. You can do this with +
```
this.error = 'Unfortunately, the language ' +
    'you requested does not have...';
```